tests/e2e/systemtest/test_noobaa_system.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.bucket_utils import (
+    write_data_to_buckets_and_check_integrity,
+    verify_s3_buckets_integrity,
+)
+
+log = logging.getLogger(__name__)
+
+
+class TestNoobaaSystem(E2ETest):
+    """
+    Test Noobaa System
+
+    """
+
+    def test_noobaa_system(
+        self,
+        bucket_factory,
+        mcg_obj,
+        awscli_pod,
+    ):
+        """
+        Test Procedure:
```

Comment:
Please add test description. For example: "The objective of this test case is to verify that noobaa can be recovered successfully by following the steps mentioned in https://access.redhat.com/solutions/5843261, when db-noobaa-db-0 PVC is accidentally corrupted/deleted on a cluster which is configured with various noobaa/mcg feature like mcg bucket replication, noobaa caching, namespace obc etc.,"